### ISSUE TITLE
Convolution Issue when converting onnx > h5

### ISSUE THREAD

**Issue by AmitAilianiSDC** [PINTO0309/onnx2tf#216]
### Issue Type

Others

### onnx2tf version number

1.7.10

### onnx version number

1.13.1

### tensorflow version number

2.13.0-dev20230223

### Download URL for ONNX

None

### Parameter Replacement JSON

```json
None
```

### Description

Getting an issue with convolution when converting onnx2tf and its using use_cudnn_on_gpu = true.

Depth of input (1) is not a multiple of input depth of filter (40) for '{{node tf.nn.convolution_18/convolution}} = Conv2D[T=DT_FLOAT, data_format="NHWC", dilations=[1, 1, 1, 1], explicit_paddings=[], padding="SAME", strides=[1, 1, 1, 1], use_cudnn_on_gpu=true](Placeholder, tf.nn.convolution_18/convolution/filter)' with input shapes: [1,38,1,1], [1,1,40,10].

Call arguments received by layer "tf.nn.convolution_18" (type TFOpLambda):
  • input=tf.Tensor(shape=(1, 38, 1, 1), dtype=float32)
  • filters=tf.Tensor(shape=(1, 1, 40, 10), dtype=float32)
  • strides=['1', '1']
  • padding='SAME'
  • data_format=None
  • dilations=['1', '1']
  • name=None
ERROR: input_onnx_file_path: /home/amita/dev/ai-model-conversion-main/ai-model-conversion/ml/models/teeth_detection/model_cxcywh_v2.onnx
ERROR: onnx_op_name: /6/se/se/1/Conv
ERROR: Read this and deal with it. https://github.com/PINTO0309/onnx2tf#parameter-replacement
ERROR: Alternatively, if the input OP has a dynamic dimension, use the -b or -ois option to rewrite it to a static shape and try again.
ERROR: If the input OP of ONNX before conversion is NHWC or an irregular channel arrangement other than NCHW, use the -kt or -kat option.
ERROR: Also, for models that include NonMaxSuppression in the post-processing, try the -onwdt option.

**Comment by PINTO0309** [comment]
That parameter is not incorporated into the tool. If you are using it with modifications, you are on your own.

**Comment by AmitAilianiSDC** [comment]
Are you referring to use_cudnn_on_gpu? I did not set this at all. How to disable it?

**Comment by PINTO0309** [comment]
I would avoid closing issues blindly. But you have not provided the minimum information necessary to merit investigation. That information alone does not tell me anything. Therefore, it was closed.

Please follow the issue template and provide the minimum information required.
https://github.com/PINTO0309/onnx2tf/blob/main/CONTRIBUTING.md#1-issues

**Comment by AmitAilianiSDC** [comment]
I am attaching the screenshot of the onnx graph where the issue is occurring. 
![image](https://user-images.githubusercontent.com/17713854/222076091-129a66b5-abed-4708-bba5-d4462183273a.png)

**Comment by AmitAilianiSDC** [comment]
I found there is an issue with the ReduceMean operation from onnx to tf. It is GlobalAveragePool. I would like  to pass these parameters for tf op Reduce mean with replacement.json. But there is no effect of passing in replacement.json during the conversion. {
  "format_version": 1,
  "operations": [
    {
      "op_name": "/6/se/se/0/GlobalAveragePool",
      "param_target": "attributes",
      "axes": [0,1],
      "keepdims": 1
    }
  ]
}

**Comment by AmitAilianiSDC** [comment]
I was able to fix the error by passing in replace.json!
